### PR TITLE
Fix the Custom node crash

### PR DESCRIPTION
**Pull request by ramramps**
### Purpose

This PR fixes the crash introduced in this PR:  https://github.com/DynamoDS/Dynamo/pull/8708

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@mjkkirschner

### CODE REVIEW

File: src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs [hunk 1]
```diff
@@ -445,8 +445,10 @@ public string InputSymbol
                         }
                     }
                 }
-
-                
+                else
+                {
+                    Parameter = new TypedParameter(name, type, defaultValue, null, comment);
```
Comment (ramramps) [inline src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs]: I don't know why self serve CI did not catch this error.

**Comment by mjkkirschner** [comment]
so under what circumstances does this else block get executed and the type turns into `var`?

**Comment by ramramps** [comment]
when an input node is created in custom node workspace.

**Comment by mjkkirschner** [comment]
👍 any thoughts on a quick test for this? Create a new blank symbol, save, check if var gets serialized?

**Comment by ramramps** [comment]
tests are already there for this. running the self serve CI with fix from Aaron for category.

**Comment by mjkkirschner** [comment]
@ramramps did this and your other PR get cherry picked yet or waiting for testing results?

**Comment by ramramps** [comment]
Neal or Sid should test this on master branch and confirm the testing, because this is a crash and regression.

**Pull request merged**